### ISSUE TITLE
Map server crash when 0 hp

### ISSUE THREAD

**Issue by MrAntares** [rathena/rathena#1839]
Client: 2012-04-10
GIT version: 2016-12-29 https://github.com/rathena/rathena/commit/a06028bb48385e32364fd309d068d52be5f43615
Renewal
No source mods

Related to this:
https://rathena.org/board/topic/108880-map-server-crash/

When an alive character data is loaded and saved with 0 hp for some anomalous reason, logging in with said character will cause a mapserver crash and render the character unusable and a server crasher on every login attempt until the data is manually edited in the database.

**Comment by Everade** [comment]
Client: 2015-09-16aRagexe
GIT version: a9faab8
Renewal

Cannot confirm.

Tested by **manually** setting the HP to 0.
The only thing that happened was that the client timed out while trying to load map server data.

It didn't bother the map server at all.

**Comment by MrAntares** [comment]
You might be right that setting the data manually to 0 won't cause the crash, but there is an issue when for some unknown reason the server starts to load characters with 0 values from the database and thst is what triggers the crash. After  I restarted the server and tried to login with the corrupted characters the client simply timed out. So it is a combination of the unknown bug and the 0 hp. After I edited the caharacter hp value everything returned to normal but the bug what caused the wole thing is still there somewhere
